package.json

```diff
@@ -53,23 +52,23 @@
   "dependencies": {
     "@hapi/boom": "^7.4.2",
     "@hapi/joi": "^15.1.0",
-    "async-iterator-last": "^1.0.0",
     "cids": "~0.7.1",
     "debug": "^4.1.0",
     "err-code": "^2.0.0",
     "hamt-sharding": "~0.0.2",
-    "interface-datastore": "~0.7.0",
+    "interface-datastore": "^0.8.0",
     "ipfs-multipart": "~0.2.0",
-    "ipfs-unixfs": "~0.1.16",
-    "ipfs-unixfs-exporter": "~0.38.0",
-    "ipfs-unixfs-importer": "~0.40.0",
+    "ipfs-unixfs": "^0.2.0",
+    "ipfs-unixfs-exporter": "^0.39.0",
+    "ipfs-unixfs-importer": "^0.42.0",
```

Comment:
Do you need the importer/exporter/unixfs with metadata support for this?

Reply:
I ask because I wouldn't pull them in until the dust has settled on https://github.com/ipfs/specs/pull/229

Reply:
No I do not

Reply:
I was just like, only forwards...

Reply:
A noble approach. Bravo, sir!

I'd, uh, still back the upgrade out of this PR though..

Reply:
Not too far forwards!